MekHQ/unittests/mekhq/campaign/autoresolve/ResolverTest.java

```diff
@@ -101,22 +99,22 @@ public enum TeamArrangement {
     }
 
     static double lowerBoundTeam1() {
-        return 0.45;
+        return 0.35;
     }
     static double upperBoundTeam1() {
-        return 0.52;
+        return 0.50;
     }
```

Comment:
How are you handling 3rd party forces, btw? For example, pirates from the handful of pirate scenario modifiers.

This isn't necessarily related to this line, more of a curiosity.

Reply:
The attacker considers an enemy target randomly, so as long as there is an enemy available it will try and pick it appart.

The chances are really bad for everyone when you add a third party.

Reply:
Awesome stuff!

Thanks for taking the time to respond :)